### ISSUE TITLE
Error for testing sample files

### ISSUE THREAD

**Issue by jackchen129** [imgag/plasmIDent#2]
Hi,

I just installed and tested the sample files. It showed an error:

ERROR ~ Argument of `file` function cannot be null

 -- Check script 'main.nf' at line: 449 or see '.nextflow.log' file for more details
Containers    : caspargross/plasmident:latest
Minimum contig length cannot be shorter then seqPadding. Please adjust parameters and restart

Could you let me know how to fix it?

Thanks.

**Comment by angelovangel** [comment]
If still relevant - may be your input sample sheet has a header? It should be without a header, the documentation is not clear here.

**Comment by caspargross** [comment]
Yes, the Input sample sheet should not contain a header, I updated the Documentation. Can you try again if this is still relevant?
